packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,17 +10,19 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
--   [**] [Embed block] Add the top 5 specific embed blocks to the Block inserter list [#34967]
 -   [*] [Embed block] Fix inline preview cut-off when editing URL [#35321]
 
-1.62.2
-------
-* Same as 1.62.1 but with the changelog.
+## 1.63.0
+-   [**] [Embed block] Add the top 5 specific embed blocks to the Block inserter list [#34967]
+-   [*] Embed block: Fix URL update when edited after setting a bad URL of a provider [#35013]
+-   [**] Users can now contact support from inside the block editor screen. [#34890]
+
+## 1.62.2
+-   Same as 1.62.1 but with the changelog.
 
-1.62.1
-------
-* [**] Image block: fix height and border regression. [https://github.com/WordPress/gutenberg/pull/34957]
-* [**] Column block: fix width attribute flout cutoff. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921]
+## 1.62.1
+-   [**] Image block: fix height and border regression. [https://github.com/WordPress/gutenberg/pull/34957]
+-   [**] Column block: fix width attribute flout cutoff. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921]
```

Comment:
I noticed that this entry has a couple of issues (described [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4031/files#r723059270)), I'm wondering if we could address them here or in a new PR, wdyt?

Reply:
I'd prefer to update them in a separate PR because to make sure this is merged as soon as possible.
If we address it in a separate PR, it probably doesn't even need a Gutenberg Mobile PR, since it's just a changelog edit.

Reply:
Ok, makes sense to me, let's not block the PR.

> If we address it in a separate PR, it probably doesn't even need a companion Gutenberg Mobile PR, since it's just a changelog edit.

Yep, although in this case, we'd require a change in GB-mobile too because the same issues are also in the GB-mobile release notes 😅 .

Reply:
I'm marking this resolved and merging the PR.